packages/server/src/resolvers.ts

```diff
@@ -119,7 +119,7 @@ export const resolvers = {
         }
       })
     },
-    createLocation: async (locationInput: Prisma.LocationCreateInput, context: Context) => {
+    /*createLocation: async (locationInput: Prisma.LocationCreateInput, context: Context) => {
```

Comment:
Is this a change you made? Cleanup if you did or just leave it if you haven't touched it

Reply:
Aman changed this during the meeting today on my laptop as the database does not work otherwise and would spit out an error